docs/application/web/get-started/overview.md

```diff
@@ -18,13 +18,29 @@ When creating your first Tizen Web application, you can select from the followin
 
     A TV application is targeted to run on smart TVs, and can be optimized for larger screens and remote control.
 
--    [Samsung Smart TV Web application](tv/first-samsung-tv-app.md)
+-   [Samsung Smart TV Web application](tv/first-samsung-tv-app.md)
 
-     A TV application is targeted to run on Samsung smart TVs, and can be optimized for larger screens and remote control.
+    A TV application is targeted to run on Samsung smart TVs, and can be optimized for larger screens and remote control.
 
 -   [Mobile Web application](mobile/first-app.md)
 
     A mobile application is targeted to run on smart phones, and can take advantage of a wide variety of functionalities available on those devices.
 
```

Comment:
Please add here!

-   [Web service application](web-service/first-service.md)
    A service is an application that runs some business logics in background, based on node.js environmnet.  A web service application is an application that runs a javascript file which contains service application life-cycle callbacks in background.

Reply:
How about this:

A service is an application that runs some business logics in background, based on node.js environmnet. A web service application consists of a javascript file which contains service application life-cycle callbacks in background.